packages/pages-components/src/components/schema/schemaTypes/FAQPage.tsx

```diff
@@ -2,15 +2,31 @@ type RTF2 = {
   json?: Record<string, any>;
 };
 
-type FAQ =
-  | {
-      question: string;
-      answer: string;
-    }
-  | {
-      question: string;
-      answerV2: RTF2;
-    };
+type FAQ = PlainTextFAQ | RichTextFAQ;
+
+type PlainTextFAQ = {
+  question: string;
+  answer: string;
+};
+
+type RichTextFAQ = {
+  question: string;
+  answerV2: RTF2;
+};
+
+const validatePlainTextFAQ = (faq: any): faq is PlainTextFAQ => {
+  if (typeof faq !== "object") {
+    return false;
+  }
+  return "question" in faq && "answer" in faq;
```

Comment:
Return the typeof check here too to make it consistent with validateRichTextFAQ? Or vice versa since other ones are more like this one.